test_test_cases.R

```diff
@@ -0,0 +1,66 @@
+library(readr)
+library(yaml)
+
+test_cases_dir <- "~/Dropbox (2° Investing)/2° Investing Team/People/Jacob/p2020_test_cases"
```

Comment:
as you identified, should aim to not hardcode this

Reply:
yeah, not sure what's best here.... probably do **not** want to add the sample cases to the repo... so it's either
1. point to somewhere we can assume our internal users will have access to
2. expect our internal users to set this path explicitly to something appropriate when running the tests
3. use some env variable or option to set a default path to look in
4. build some complex UI that asks the user for the path whenever it is run

not thrilled about any of those options... but out of those, I kinda prefer 2.

Reply:
:+1: for item 2, maybe with a default like item 1.

Reply:
so that's essentially what we have... we expect users to set this path to something appropriate for them... but by default we put in a path that is likely to work for some of our internal users... and I added a test to see if the directory exists and fails fast if it doesn't 65dfcaf439eca39f5708b8ccfdc35e565d85fc33